azerothcore/azerothcore-wotlk
# fix(DB/Quest): Take No Chances

## CHANGES PROPOSED:
- Grain should be burned only once
-  


## ISSUES ADDRESSED:
- Closes #2930 



## TESTS PERFORMED:
- Tested in game
-



## HOW TO TEST THE CHANGES:

- .go cre 97893
- .q add 11913
- use the quest item [Wendy's Torch] for burning and see that you won't be able to burn it endlessly

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
